elasticsearch/prep-install.origin

```diff
@@ -7,6 +7,18 @@ if [ -z "${ES_CLOUD_K8S_URL:-}" ] ; then
     ES_CLOUD_K8S_URL=io.fabric8/elasticsearch-cloud-kubernetes/${ES_CLOUD_K8S_VER}
 fi
 if [ -z "${OSE_ES_URL:-}" ] ; then
-    OSE_ES_URL=io.fabric8.elasticsearch/openshift-elasticsearch-plugin/${OSE_ES_VER}
+    MANPATH="" source /opt/rh/rh-maven33/enable
+    set +e
+    mvn dependency:get -Dartifact=io.fabric8.elasticsearch:openshift-elasticsearch-plugin:${OSE_ES_VER}
```

Comment:
This spews a lot of output . . . not sure where we can put it.  It doesn't look good in the CI output.

Reply:
maven doesnt seem to have a 'search' feature.  I am really not fond of this approach but its the only way i have found to determine if it is published.

Reply:
understood - it is distressing to look at the CI log from this run and see hundreds of red lines of `[ERROR]` . . .